src/app/components/datepicker/datepicker.component.html

```diff
@@ -0,0 +1,5 @@
+<mat-form-field>
+  <input matInput [matDatepicker]="dp" placeholder="dd.MM.yyyy" [formControl] ="date" [value]="dateForParent">
+  <mat-datepicker-toggle matSuffix [for]="dp"></mat-datepicker-toggle>
+  <mat-datepicker #dp ></mat-datepicker>
```

Comment:
please avoid cutting of words for clearer code

Reply:
changed